packages/gen-ai/frontend/src/app/utilities/const.ts

```diff
@@ -1,18 +1,10 @@
 import { asEnumMember, DeploymentMode } from 'mod-arch-core';
 
 const URL_PREFIX = process.env.URL_PREFIX || '/gen-ai';
```

Comment:
Shouldn't we update this as well?

Reply:
No, as this is used for apis and frontend url prefix is different from api url prefix